README.md

```diff
@@ -11,16 +11,42 @@
 ## First, get a copy of Raylib
 You can get the library by compiling it from the [source](https://github.com/raysan5/raylib), download the [official precompiled binaries](https://github.com/raysan5/raylib/releases). The local copies of binaries are no longer available, as that history was lost.
 
-*WARNING*: Make sure you get the correct copy of the raylib library based on the version of raylib-d! Getting the incorrect version will cause SILENT compatibility errors, including memory corruption.
+#### Linux:
+
+You must make raylib visible to the linkage binder. CD into the extracted raylib folder.
+
+Now we must make raylib visible to the compiler and linker system wide. Simply run the following.
+```
+sudo mv include/* /usr/include/
+
+sudo mv lib/* /usr/lib/
+```
+
+#### Windows:
+On Windows you must drag and drop all the required library files into the root directory of your project. These are `raylib.dll`, `raylib.lib`, and `raylibdll.lib`.
+
+*WARNING*: Make sure you get the correct copy of the raylib library based on the version of raylib-d! Getting the incorrect version will cause SILENT compatibility errors, including memory corruption. It is extremely important to match these together.
```

Comment:
This warning should be at the top, where it was. It's not specific to Windows.

Reply:
> 

Fixed!

Reply:
> This warning should be at the top, where it was. It's not specific to Windows.

Fixed!